yapapi/ctx.py

```diff
@@ -459,16 +473,31 @@ def commit(self, timeout: Optional[timedelta] = None) -> Work:
         self._pending_steps = []
         return Steps(*steps, timeout=timeout)
 
-    async def get_usage(self):
+    async def get_raw_usage(self) -> yaa_ActivityUsage:
+        """Get the raw usage vector for the activity bound to this work context.
+        The value comes directly from the low level API and is not interpreted in any way."""
```

Comment:
For a multi-line docstring, there should be a blank line after the first (summary) line, see https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings.

Also I think the prevailing convention in `yapapi` docstrings is to put the closing `"""` on a new line in multi-line docstrings.